google-cloud-spanner/src/main/java/com/google/cloud/spanner/spi/v1/GapicSpannerRpc.java

```diff
@@ -875,6 +880,48 @@ private void acquireAdministrativeRequestsRateLimiter() {
     return new Paginated<>(response.getInstanceConfigsList(), response.getNextPageToken());
   }
 
+  @Override
+  public OperationFuture<InstanceConfig, CreateInstanceConfigMetadata> createInstanceConfig(
+      String parent,
+      String instanceConfigId,
+      InstanceConfig instanceConfig,
+      @Nullable Boolean validateOnly)
```

Comment:
Is there a difference between null and false in this case? If not, I would prefer using a boolean instead of a Boolean.

Reply:
If the user does not pass in any validateOnlyOption, this value will be null. Hence, we have @Nullable here. If we have this value is null, we need not to populate validateOnly field to the backend.

Reply:
Makes sense to me.